nexus_constructor/json/load_from_json_utils.py

```diff
@@ -212,33 +199,22 @@ def _create_stream(json_object: Dict) -> Stream:
             shape=shape,
         )
     if writer_module == WriterModules.NS10.value:
-        return NS10Stream(source=source, topic=topic)
+        return NS10Stream(parent_node=None, source=source, topic=topic)
```

Comment:
Remember to take care of `_create_stream` in a follow up ticket.

Reply:
Fixing it in this PR as I think it should be quite straightforward